client/src/Edit/elements/ReferenceInput.tsx

```diff
@@ -68,6 +68,8 @@ type Props = FieldProps<ReferenceType> & {
   required?: boolean;
   placeholder?: string;
   allowAbsoluteRefs?: boolean;
+  validationRegex?: string;
```

Comment:
Why should we be able to add a custom regex? I would expect that the field should know what has to validate?